tests/test_activations.py

```diff
@@ -40,6 +40,10 @@ def test_get_activation(self):
         get_activation("gelu_new")
         get_activation("gelu_fast")
         get_activation("gelu_python")
+        get_activation("quick_gelu")
```

Comment:
Just to clarify that these don't test the activations, but just check it can load those.

I presume if we have an activation it's because it's already used in at least one model and thus is tested. I have just not tested that this is the case. But we can deal with that in another PR.

Reply:
> Just to clarify that these don't test the activations, but just check it can load those.

Yes, I have just added these four loading tests for the sake of completeness as they've been missing in the `def test_get_activation(self)`.

According to the `tests/test_activations.py`, only a few `GELU` versions are "properly tested" against the official pytorch implementations. And it seems that all other activations are either using or are composed of the official pytorch functions so I assumed they are fine.

> I presume if we have an activation it's because it's already used in at least one model and thus is tested as part of the model. I have just not tested that this is the case always - hence the suggestion to simply grep if each one of them is in at least one model. But we can deal with that in another PR.

I have just checked, and it looks like they all appear somewhere in the repository.

Reply:
great! Thanks a lot for researching this and making sure we have the coverage, @eldarkurtic!